guide/daily-spending-wallet/requesting.md

```diff
@@ -189,10 +189,20 @@ A payment request with only standard data, like an amount and date, communicates
    height = 541
 %}
 
-Users should have the option to attach a note for record-keeping and for the sender to read, tags, labels and any other metadata that may be relevant to give more context to the payment.
+Users should have the option to attach a note for record-keeping and for the sender to read, tags, labels and any other meta data that may be relevant to give more context to the payment.
```

Comment:
Metadata as one word seems to be the Merriam Webster way to write this: https://www.merriam-webster.com/dictionary/metadata

Reply:
One word is fine. I changed it to make it consistent with other usages of "meta data" on the page. A quick search reveals that we spell the word inconsistently across the Guide. Perhaps I should add a "fix the metadata" PR.

Reply:
I fixed the spelling on this page in [8733a8a](https://github.com/BitcoinDesign/Guide/pull/774/commits/8733a8a836c4f30cd69e0f3761c11f0390dd901d)